src/Microsoft.AspNet.Proxy/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-*",
+  "description": "Proxy Library",
+
+  "dependencies": {
+    "Microsoft.AspNet.Http.Abstractions": "1.0.0-*",
+    "Microsoft.Framework.NotNullAttribute.Sources": {
+      "version": "1.0.0-*",
+      "type": "build"
+    }
+  },
+
+  "frameworks": {
+    "dnx451": {
+      "frameworkAssemblies": { "System.Net.Http": "" }
+    },
+    "dnxcore50": {
+      "dependencies": {
+        "System.Collections": "4.0.10-*",
+        "System.Linq": "4.0.0-*",
+        "System.Threading": "4.0.10-*",
+        "Microsoft.CSharp": "4.0.0-*",
+        "Microsoft.Net.Http.Client": "1.0.0-*"
```

Comment:
For HttpClient to work on CoreClr on non-Windows you need to do the following:
https://github.com/aspnet/dnx/blob/8d8e5611b5789a733987dfbe6f710d46c8ff026a/src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs#L45-L49